### PR TITLE
Rename module

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

**Comment by kaelig** [comment]
Note that some global variables were renamed but they couldn't be overridden so I assume nobody was using them and so are safe to rename.

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Silent mode off: `.o-ft-buttons--inverse`
 ### Deprecated
 
 The `--faux` modifier is now deprecated and will be removed in next major release.
-The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `..o-ft-buttons-pagination` instead.
+The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `.o-ft-buttons-pagination` instead.
```
Comment (AlbertoElias) [inline README.md]: .o-buttons-pagination?

Comment (kaelig) [inline README.md]: I thought I'd rename class names in the next major version.

Unless the two (`o-ft-buttons` and `o-buttons` should coexist)?

Comment (AlbertoElias) [inline README.md]: It will be quite weird if you rename everything except for classes

Comment (kaelig) [inline README.md]: Well it's easy to have both `o-ft-` and `o-` variables cohabit, but classes and mixins are a lot of overhead.

Comment (AlbertoElias) [inline README.md]: It's always good to add the new feature (in this case, new class names and mixins) before a major release so products can start adapting to the new version and the major release will be smoother.

Comment (kaelig) [inline README.md]: Alright, here it is: 87e5d96

That's going to be a lot of code. I'll release a major version right after it with much leaner code.

**Comment by AlbertoElias** [comment]
LGTM. You should get your forms update to use this new version. I'll do o-tabs

**Pull request merged**